pyeudiw/sd_jwt/sd_jwt.py

```diff
@@ -138,7 +138,11 @@ def _verify_iat(payload: dict) -> None:
 
 def _verify_key_binding(token_without_hkb: str, sd_hash_alg: str, hkb: DecodedJwt, challenge: VerifierChallenge):
     _verify_challenge(hkb, challenge)
-    _verify_sd_hash(token_without_hkb, sd_hash_alg, hkb.payload.get("sd_hash", ""))
+    _verify_sd_hash(
+        token_without_hkb, 
+        sd_hash_alg, 
+        hkb.payload.get("_sd_hash", hkb.payload.get("sd_hash", ""))
```

Comment:
is there a typo in this line?
I think it should be
```
        hkb.payload.get("_sd_hash")
```

Reply:
`hash_alg`

https://www.ietf.org/archive/id/draft-fett-oauth-selective-disclosure-jwt-02.html#section-5.2-3

Reply:
no, it is `sd_hash`

Reply:
https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-13.html#section-4.3-3.2.1